### PR TITLE
[Spec] Spring-cleaning of the test harness

### PR DESCRIPTION
**Pull request by mrackwitz**

### CODE REVIEW

**Comment by yanndupuy** [comment]
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  dupuyyann@gmail.com
    Domain zyapps.com has exceeded the max emails per hour (422/400 (105%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o1.sgmail.github.com ([192.254.114.176]:10171)
	by serveur.ex2-p10.com with esmtps (TLSv1.2:ECDHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.87)
	(envelope-from <bounces+848413-8f06-yann.dupuy=zyapps.com@sgmail.github.com>)
	id 1avrxV-001Gt0-Rd
	for yann.dupuy@zyapps.com; Thu, 28 Apr 2016 21:50:47 +0200
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed; d=github.com; 
	h=from:reply-to:to:cc:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=9q8dcPZuP1RaHDMyKGY53uaD3cI=; b=VP8hk8px2NmmLpl7
	pF1LdoE2fO2EICY9qMFrG8BWh9rga0I+TK2a1iO26t/FnGL3F+02ubGNW/sCFc+s
	Lx7IdPz35WAkMZGOzQ4nsni0QDJuxmoHFLiMOCyaardfZPmTFhfZkXrrekjpmwES
	TZlv0TEvK0WukFNfAbNDMqXRKUY=
Received: by filter0559p1mdw1.sendgrid.net with SMTP id filter0559p1mdw1.2158.572269683D
        2016-04-28 19:50:00.947524809 +0000 UTC
Received: from github-smtp2a-ext-cp1-prd.iad.github.net (github-smtp2a-ext-cp1-prd.iad.github.net [192.30.253.16])
	by ismtpd0004p1iad1.sendgrid.net (SG) with ESMTP id qSNnlzyVTlC9Xdt89vbVXg
	for <yann.dupuy@zyapps.com>; Thu, 28 Apr 2016 19:50:00.809 +0000 (UTC)
Date: Thu, 28 Apr 2016 12:49:53 -0700

**Comment by mrackwitz** [comment]
@orta: This lets Danger prevent disabling tests. Does that look right for you? How would I validate that?

File: spec/unit/installer/analyzer_spec.rb [hunk 1]
```diff
@@ -564,15 +564,6 @@ module Pod
         ]
       end
 
-      xit 'removes the specifications of the changed pods to prevent confusion in the resolution process' do
```
Comment (mrackwitz) [inline spec/unit/installer/analyzer_spec.rb]: @segiddins: I assume this is not relevant anymore. Can you confirm that?

**Comment by segiddins** [comment]
👍🏻

File: spec/unit/installer/analyzer_spec.rb (hunk 1 above)
Comment (segiddins) [inline spec/unit/installer/analyzer_spec.rb]: pretty sure that was there before I ever started touching the analyzer

**Comment by orta** [comment]
That looks good to me, you can test it locally by running `danger local` which will run the current Dangerfile against the last merged PR in the repo's commit history.

**Comment by orta** [comment]
Docs for Danger are still not a huge priority yet, so discoverability of this stuff is pretty bad

**Pull request merged**